startcloud.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
nitpick: you could have created these startup files as 1, receiving different parameters for each environment. respecting the dry pattern

Reply:
LGTM

Reply:
makes perfect sense, I could have a `./start.sh localdev` or `./start.sh cloud` to facilitate executions, thanks for the review, I'll adjust